From 4490463513be2144a6b5d87c35115f21d2eafabb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jonas=20Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Mon, 17 Feb 2020 14:21:51 +0100
Subject: [PATCH] appDisplay: Remove the top padding from folder app grid

There already is a bottom padding defined for the name container of the
folder, so remove the top padding of the appGrid and make sure we can
show a few more icons inside the folder.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/1011
---
 js/ui/appDisplay.js | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/js/ui/appDisplay.js b/js/ui/appDisplay.js
index f5e18d2ad2..17d69ced9c 100644
--- a/js/ui/appDisplay.js
+++ b/js/ui/appDisplay.js
@@ -1911,6 +1911,8 @@ var AppFolderDialog = GObject.registerClass({
             contentBox.get_width(),
             contentBox.get_height() - entryBoxHeight - spacing);
 
+        this._view._grid.topPadding = 0;
+
         super.vfunc_allocate(box, flags);
 
         // We can only start zooming after receiving an allocation
-- 
2.24.1

